src/parallelization/nesting/commonstepper.h

```diff
@@ -101,7 +101,7 @@ class CommonStepper : public Stepper<CELL_TYPE>
     Region<DIM> kernelFraction;
     bool enableFineGrainedParallelism;
 
-    inline void notifyPatchAccepters(
+    virtual inline void notifyPatchAccepters(
```

Comment:
ah, no, actually not, I need it to be a virtual function, because i override it in hpxstepper, but it is called from within the vanillastepper.